pkg/crc/config/settings.go

```diff
@@ -75,8 +76,21 @@ func RegisterSettings(cfg *Config) {
 		"Disable update check (true/false, default: false)")
 	cfg.AddSetting(ExperimentalFeatures, false, ValidateBool, SuccessfullyApplied,
 		"Enable experimental features (true/false, default: false)")
-	// shared dir is not implemented for windows yet
-	if runtime.GOOS == "darwin" || runtime.GOOS == "linux" {
+
+	// Shared directories configs
+	if runtime.GOOS == "windows" {
+		validateSmbPasswordSet := func(value interface{}) (bool, string) {
```

Comment:
Why is this not a regular named function inside a `_windows.go` file?

Reply:
Its following the same pattern that's in the config package.. and we can't use a `_windows.go` in there without a lot of refactor because how we are registering the settings

Reply:
so it is impossible to create a file named `settings_windows.go` and place this inside as a named function ?

Reply:
yeah looks like, if we do that we have to duplicate all of the common settings in `settings_windows.go`

Reply:
would prefer to do this refactor separate from this PR :) moved the function definition near the other similar functions, hopefully the code looks a bit better then before